lib/mosaic-routing/src/main/java/org/eclipse/mosaic/lib/routing/graphhopper/profile/BikeProfile.java

```diff
@@ -33,8 +31,13 @@ public class BikeProfile extends RoutingProfile {
 
     public BikeProfile() {
         super(NAME,
-                VehicleSpeed.create(NAME, 4, 2.0, false),
-                VehiclePriority.create(NAME, 4, PriorityCode.getFactor(1), false)
+                /*
+                 * Defines an encoding for speed limits. It uses just 4 bits to store the actual speed limit defined on an edge.
+                 * It divides the actual speed limit by 2 before encoding it, reducing accuracy, but increasing the maximum storable
+                 * speed limit. Therefore, the maximum speed limit to be stored is 30 km/h. (2^4-1 = 15 * 2 = 30)
+                 * These are the default values used by GraphHopper, e.g., see DefaultImportRegistry line 299.
```

Comment:
This is nitpicky, but when looking at the `DefaultImportRegistry`for graphhopper 10.2 it points to the ferry encoding.
For bikes it's here https://github.com/graphhopper/graphhopper/blob/d8a531a963ce874276af4aa6a4a1f4383ea9b3fa/core/src/main/java/com/graphhopper/routing/ev/DefaultImportRegistry.java#L291C1-L291C109